homeassistant/components/wallbox/sensor.py

```diff
@@ -34,10 +33,12 @@ class WallboxSensor(CoordinatorEntity, SensorEntity):
     def __init__(self, coordinator, idx, ent, config):
         """Initialize a Wallbox sensor."""
         super().__init__(coordinator)
-        self._properties = CONF_SENSOR_TYPES[ent]
-        self._name = f"{config.title} {self._properties[CONF_NAME]}"
-        self._icon = self._properties[CONF_ICON]
-        self._unit = self._properties[CONF_UNIT_OF_MEASUREMENT]
+        _properties = CONF_SENSOR_TYPES[ent]
```

Comment:
Do use _ for method variables, that is reserved for internal class variables.

Apart from that, it does not make the code more readable (I prefer to reference CONF_SENSOR_TYPES directly, in __init__), but that is a matter of taste.

Reply:
I took your suggestions and referred directly to the original dict.